ui/src/testing/test-helpers.ts

```diff
@@ -64,3 +83,29 @@ export function findElementsReact<C>(
 ) {
   return [].slice.call(fixture.debugElement.nativeElement.querySelectorAll(selector));
 }
+
+export const signedInDependencies = {
```

Comment:
Thanks for factoring this out - this is nice.

nit: Is there a reason not to just define normal Angular module and import it into your tests?